development/tests/test_match_result_text.py

```diff
@@ -0,0 +1,153 @@
+from unittest.mock import patch
+
+from django.test import TestCase
+
+from development.common.match_result_text import generate_text
+
+
+class TestMatchResultText(TestCase):
+
+    '''
+    'action': 'SHOOT', 'data': '{"from_row": 8, "from_col": 0, "direction": "NORTH",
```

Comment:
Is this used in any way? if not it should be removed

Reply:
Working on it. I left it by accident, sorry.